### PR TITLE
add-tests

### PR DESCRIPTION
**Pull request by TheRealHaui**
Created tests to cover previously uncovered code.

### CODE REVIEW

File: graphjet-core/src/test/java/com/twitter/graphjet/bipartite/segment/LeftIndexedBipartiteGraphSegmentTest.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+/**
+ * Unit tests for class {@link LeftIndexedBipartiteGraphSegment}.
+ *
+ * @date 2017-07-28
+ * @see LeftIndexedBipartiteGraphSegment
+ **/
+public class LeftIndexedBipartiteGraphSegmentTest {
+
+  @Test
+  public void testOne() {
```
Comment (jerryjiangabc) [inline graphjet-core/src/test/java/com/twitter/graphjet/bipartite/segment/LeftIndexedBipartiteGraphSegmentTest.java]: change testOne and testTwo to something more specific.

File: graphjet-core/src/test/java/com/twitter/graphjet/algorithms/RecommendationStatsTest.java [hunk 2]
```diff
@@ -0,0 +1,79 @@
+package com.twitter.graphjet.algorithms;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Unit tests for class {@link RecommendationStats}.
+ *
+ * @date 2017-07-28
```
Comment (jerryjiangabc) [inline graphjet-core/src/test/java/com/twitter/graphjet/algorithms/RecommendationStatsTest.java]: could you remove the date here and everywhere else?

File: graphjet-core/src/test/java/com/twitter/graphjet/bipartite/segment/LeftIndexedBipartiteGraphSegmentTest.java (hunk 1 above)
Comment (TheRealHaui) [inline graphjet-core/src/test/java/com/twitter/graphjet/bipartite/segment/LeftIndexedBipartiteGraphSegmentTest.java]: Done.

File: graphjet-core/src/test/java/com/twitter/graphjet/algorithms/RecommendationStatsTest.java (hunk 2 above)
Comment (TheRealHaui) [inline graphjet-core/src/test/java/com/twitter/graphjet/algorithms/RecommendationStatsTest.java]: Done.

**Comment by TheRealHaui** [comment]
Done.

**Pull request merged**